RELICENSING.md

```diff
@@ -10,7 +10,7 @@ relicensed to the Apache License, Version 2.0, and are submitted pursuant to
 the Developer Certificate of Origin, version 1.1:
 
 - Larry Gritz (@lgritz)
-
+- Daniel Wyatt
 
```

Comment:
```suggestion
- Dalai Felinto
```

Reply:
There were not enough extra lines so I may have done this wrong, but here it is (thank you Brecht for the poke).